tools/python/copy-table/README.md

```diff
@@ -0,0 +1,120 @@
+# Timestream Unload & BatchLoad Automation
```

Comment:
This is from upstream mainline, when our mainline can be synced, these changes will disappear.

Reply:
Our mainline will not be synced, instead I have created a `develop` branch.